packages/TelegramClientTests-Core.package/TCTCMessageTests.class/instance/testMessageShouldNotNotify.st

```diff
@@ -0,0 +1,10 @@
+testing
+testMessageShouldNotNotify
+
+ 	| chat message |
+
+ 	chat := TCCChat newFromChatEvent: TCTMMocks mockMutedChatEvent.
+
+ 	message := TCCMessage newFromMessageEvent: (TCTMMocks mockMessageJsonFrom: chat id) in: chat with: self core.
+
+ 	self assert: false equals: message shouldNotify.
```

Comment:
Maybe we could use a partial test function to remove the code duplication in testMessageShouldNotify

Reply:
please check if the refactoring fixes the issue